Sources/Bow/Transformers/StoreT.swift

```diff
@@ -1,39 +1,77 @@
+/// Witness for the `StoreT<S, W, A>` data type. To be used in simulated Higher Kinded Types.
 public final class ForStoreT {}
+
+/// Partial application of the `StoreT` type constructor, omitting the last parameter.
 public final class StoreTPartial<S, W>: Kind2<ForStoreT, S, W> {}
+
+/// Higher Kinded Type alias to improve readability of `Kind<StoreTPartial<S, W>, A>`.
 public typealias StoreTOf<S, W, A> = Kind<StoreTPartial<S, W>, A>
 
+/// Witness for the `Store<S, A>` data type. To be used in simulated Higher Kinded Types.
 public typealias ForStore = ForStoreT
+
+/// Partial application of the `Store` type constructor, omitting the last parameter.
 public typealias StorePartial<S> = StoreTPartial<S, ForId>
+
+/// Higher Kinded Type alias to improve readability of `Kind<StorePartial<S>, A>`.
 public typealias StoreOf<S, A> = StoreTOf<S, ForId, A>
+
+/// The Store type is equivalent to the StoreT type, with the base Comonad being Id.
 public typealias Store<S, A> = StoreT<S, ForId, A>
 
+/// The Store Comonad Transformer. This Comonad Transformer extends the context of a value in the base Comonad so that it depends on a position of the type of the state.
```

Comment:
```suggestion
/// The State Comonad Transformer. This Comonad Transformer extends the context of a value in the base Comonad so that it depends on a position of the type of the state.
```

Reply:
I think it is useful to know the dual monad

Reply:
That is reflected in `Pairing`, in PR #552, for this and the other Comonad Transformers.

Reply:
`CoWriter` - Traced
`CoReader` - Env